decidim-comments/app/types/decidim/comments/comment_type.rb

```diff
@@ -11,7 +11,9 @@ module Comments
       field :body, !types.String, "The comment message"
 
       field :createdAt, !types.String, "The creation date of the comment" do
-        property :created_at
+        resolve lambda { |obj, _args, _ctx|
+          obj.created_at.to_time.iso8601
```

Comment:
I'm not really sure... I found that as an example but I think I can use the `iso8601` method directly.